LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (C) 2016 Crown Copyright (Government Digital Service)
```

Comment:
I think we usually backdate the year on the licence, see https://github.com/alphagov/static/commit/114ce2a32333d0bb10b2747c92a795faf3dbedb0